website/src/content/docs/common/service/NodePort.md

```diff
@@ -72,4 +72,17 @@ service:
         protocol: http
         targetPort: 8080
         nodePort: 30080
+    integrations:
+      traefik:
+        enabled: true
+        forceTLS: true
+        insecureSkipVerify: false
+        serverName: "my.service.com"
+        rootCAs:
+          - configMapRef:
+              name: configmap-name
+              expandObjectName: false
+          - secretRef:
+              name: secret-name
+              expandObjectName: true
```

Comment:
We dont want integrations listed on every possible service type, use dedicated integration pages for it

Reply:
Removed.